tekton/tasks/qeb-hwt-task.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: qeb-hwt
+  namespace: default
+spec:
+  inputs:
+    resources:
+      - name: git-repo # target path is /workspace otherwise targetPath: specific needs to be included
+        resourceSpec:
+          type: git
+          description: "The input is code from a git repository"
+
+  outputs:
+    resources:
+      - name: justification
+        type: string
+        description: "Justification from adviser run"
+  steps:
+    - name: thamos-advise
+      image: "quay.io/pacospace/thamos:v0.8.1-dev"  # image need to contain python
+      script: |
+        #!/usr/bin/env python3
+        import os
+        import json
+        import yaml
+        import logging
+        from pathlib import Path
+
+        from thamos.lib import advise_here
+        from thamos.config import config
+
+        _LOGGER = logging.getLogger("thoth.qebhwt")
+
+        # os.chdir("/workspace")
+        config.explicit_host = "{{inputs.parameters.THOTH_HOST}}"
```

Comment:
Did you test this `inputs.parameters.THOTH_HOST` pattern, i feel it would depend upon git/pr payload which tekton would receive.

Reply:
Right, actually this has to be changed to `params`. In current implementation it is in the Openshift template: https://github.com/thoth-station/Qeb-Hwt/blob/2c4d41943211b2f9e2464bd69e43c2ab760d5f9c/openshift/qeb-hwt-template.yaml#L53